lib/protocol/routes.js

```diff
@@ -581,7 +581,7 @@ const METHOD_MAP = {
       // We want to either a value (old JSONWP) or a text (new W3C) parameter,
       // but only send one of them to the command (not both).
       // Prefer 'value' since it's more backward-compatible.
-      makeArgs: (jsonObj) => [jsonObj.value || jsonObj.text],
+      makeArgs: (jsonObj) => (jsonObj.value || jsonObj.text) == null ? [''] : [jsonObj.value || jsonObj.text],
```

Comment:
`[jsonObj.value || jsonObj.text || '']` should work

Reply:
or even better `[jsonObj.value ?? jsonObj.text]`

Reply:
Hi Mykola, 
I chosed the first change, if the `value` and `text` are both null value, the null value could also send to device.
so it seems first chose is better.

Regards

Reply:
then it would make sense to still replace `||` with `??`

Reply:
@mykola-mokhnach finished the change, please kindly review, Thank you.